### PR TITLE
Build multiple platform images by Github Action buildin Steps

### PR DESCRIPTION
**Pull request by panpan0000**
to address https://github.com/cloudtty/cloudtty/issues/41

Thanks to @barnettZQG

I'm not familiar with github action, and there's no confident about this code , so I keep the old `make` command by purpose until the new code has run well.

### CODE REVIEW

File: .github/workflows/image-release.yaml [hunk 1]
```diff
@@ -25,3 +25,32 @@ jobs:
...
+          file: docker/Dockerfile-webtty
+          labels: |-
+            org.opencontainers.image.source=https://github.com/${{ github.repository }}
+            org.opencontainers.image.revision=${{ github.sha }}
+          platforms: linux/amd64,linux/arm64
+          #push: ${{ github.event_name != 'pull_request' }}
+          build-args: |
+            VERSION=${{ steps.get_version.outputs.VERSION }}
+          tags: |-
+             ghcr.io/cloudtty/cloudshell:${{ steps.get_version.outputs.VERSION }}
```
Comment (barnettZQG) [inline .github/workflows/image-release.yaml:41]: Here miss the get_version step and can not get the VERSION from the outputs.

File: .github/workflows/image-release.yaml [hunk 2]
```diff
@@ -25,3 +25,32 @@ jobs:
       - name: Push 
         run: |
               make docker-push
+      - name: Build & Pushing cloudshell image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile-webtty
```
Comment (barnettZQG) [inline .github/workflows/image-release.yaml:32]: You should change the Dockerfile to install the multiple platform ttyd (arm64 and amd64)

File: .github/workflows/image-release.yaml (hunk 1 above)
Comment (panpan0000) [reply .github/workflows/image-release.yaml:50]: fixed

File: .github/workflows/image-release.yaml (hunk 2 above)
Comment (panpan0000) [reply .github/workflows/image-release.yaml:41]: fixed. linux command `arch` seems helps a lot !

**Comment by panpan0000** [comment]
Thank you  @barnettZQG , comments addressed

**Comment by panpan0000** [comment]
I will merge it if no objection.

**Pull request merged**